autosave/mixins.py

```diff
@@ -83,17 +83,16 @@ def autosave_js(self, request, object_id, extra_context=None):
         except FieldDoesNotExist:
             raise
 
-        if not object_id:
-            autosave_url = reverse("admin:%s_%s_add" % info)
-            add_log_entries = LogEntry.objects.filter(
-                    user=request.user,
-                    content_type=ContentType.objects.get_for_model(self.model),
-                    action_flag=ADDITION)
-            try:
-                updated = add_log_entries[0].action_time
-            except IndexError:
-                pass
-        else:
+        if object_id:
```

Comment:
Actually you can just get rid of this conditional, since this method won't ever be called anymore if there isn't an `object_id` (that only happened when the media was added on the `add_view`, which we're no longer doing.